modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/AgentUtils.java

```diff
@@ -298,7 +301,16 @@ public static String nid8(Collection<UUID> nids) {
      * @throws Exception If failed to send event.
      */
     public static void send(Session ses, WebSocketEvent evt) throws Exception {
-        ses.getRemote().sendStringByFuture(toJson(evt)).get();
+        Future<Void> fut = ses.getRemote().sendStringByFuture(toJson(evt));
+
+        try {
+            fut.get(10L, TimeUnit.SECONDS);
```

Comment:
Are you sure that 10 seconds in enough? May be 30 seconds or 1 minutes will be better?

Reply:
We need test this under load and increase if needed. Or separate timeouts for user actions and other events.